### PR TITLE
"Adding file for Lesson2GridPrinter

### PR DESCRIPTION
**Pull request by Jaidjen**

### CODE REVIEW

File: students/Jaidjen/lesson2/Lesson2FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for number in range(1,101):
+    if number % 3 is 0:
```
Comment (natasha-aleksandrova) [inline students/Jaidjen/lesson2/Lesson2FizzBuzz.py]: use `is` only with objects, for example `is True` `is None`

for primitive values comparison use `==`

File: students/Jaidjen/lesson2/Lesson2Fibonacci.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        return(recur_fibo(n-1)+recur_fibo(n-2))
+
+nterms =10
+
+if nterms <=0:
+    print("Please enter a positive integer")
+else:
+    print("Fibonacci sequence:")
+    for i in range(nterms):
+        print(recur_fibo(i))
```
Comment (natasha-aleksandrova) [inline students/Jaidjen/lesson2/Lesson2Fibonacci.py]: looks like you're still working on this exercise

**Comment by Jaidjen** [comment]
Thank you! 
Yes, I am still working on the Fibonacci one. 
Do I need to wait until I’m done before adding the file?

**Comment by natasha-aleksandrova** [comment]
no worries - we just have to clarify in situations like this

**Comment by Jaidjen** [comment]
Thank you!

**Pull request merged**